### PR TITLE
enable binary body

### LINKED ISSUE #30: Body as List<int> not supported
Currently it's not possible to send a List<int> in the body directly, it's useful when you want to upload a file directly as binary data.

### PR DESCRIPTION
**Pull request by jaumard**
fix #30

### CODE REVIEW

**Comment by jaumard** [comment]
A new deployment will need to be done for resty, retrofit and retrofit_gen. 
Retrofit need new resty in his pubspec
RetrofitGen need new Resty and Retrofit in his pubspec

File: resty/lib/src/jaguar_resty_base.dart [hunk 1]
```diff
@@ -495,7 +495,8 @@ class Post extends RouteBase implements RouteWithBody {
 
   Post cookies(List<ClientCookie> cookies) => super.cookies(cookies);
 
-  Post body(String body) {
+  Post body(dynamic body) {
```
Comment (tejainece) [inline resty/lib/src/jaguar_resty_base.dart]: For type safety, can we keep `Post body(String body)` as it is and add `Post bytes(String body)` or `Post bodyBytes(String body)`?

Comment (jaumard) [reply resty/lib/src/jaguar_resty_base.dart]: As you want here, I'm personally don't mind having dynamic here as I assert the type on the method :)

Comment (tejainece) [reply resty/lib/src/jaguar_resty_base.dart]: Nice. resty can be used directly without retrofit.

Comment (jaumard) [reply resty/lib/src/jaguar_resty_base.dart]: let's go for bytes :)

**Pull request merged**